circuits/cpp/src/aztec3/utils/types/circuit_types.hpp

```diff
@@ -1,68 +1,68 @@
 #pragma once
-#include <barretenberg/stdlib/primitives/address/address.hpp>
-#include <barretenberg/stdlib/encryption/schnorr/schnorr.hpp>
+#include "native_types.hpp"
+
+#include <barretenberg/stdlib/commitment/pedersen/pedersen.hpp>
 #include <barretenberg/stdlib/encryption/ecdsa/ecdsa.hpp>
+#include <barretenberg/stdlib/encryption/schnorr/schnorr.hpp>
+#include <barretenberg/stdlib/hash/blake2s/blake2s.hpp>
+#include <barretenberg/stdlib/hash/pedersen/pedersen.hpp>
+#include <barretenberg/stdlib/primitives/address/address.hpp>
 #include <barretenberg/stdlib/primitives/bigfield/bigfield.hpp>
 #include <barretenberg/stdlib/primitives/biggroup/biggroup.hpp>
 #include <barretenberg/stdlib/primitives/bit_array/bit_array.hpp>
 #include <barretenberg/stdlib/primitives/bool/bool.hpp>
 #include <barretenberg/stdlib/primitives/byte_array/byte_array.hpp>
+#include <barretenberg/stdlib/primitives/curves/bn254.hpp>
+#include <barretenberg/stdlib/primitives/group/group.hpp>
 #include <barretenberg/stdlib/primitives/packed_byte_array/packed_byte_array.hpp>
-#include <barretenberg/stdlib/primitives/uint/uint.hpp>
 #include <barretenberg/stdlib/primitives/point/point.hpp>
-#include <barretenberg/stdlib/primitives/group/group.hpp>
-#include <barretenberg/stdlib/primitives/curves/bn254.hpp>
-#include <barretenberg/stdlib/recursion/verifier/verifier.hpp>
+#include <barretenberg/stdlib/primitives/uint/uint.hpp>
 #include <barretenberg/stdlib/recursion/verification_key/verification_key.hpp>
 #include <barretenberg/stdlib/recursion/verifier/program_settings.hpp>
-#include <barretenberg/stdlib/commitment/pedersen/pedersen.hpp>
-#include <barretenberg/stdlib/hash/pedersen/pedersen.hpp>
-#include <barretenberg/stdlib/hash/blake2s/blake2s.hpp>
-#include "native_types.hpp"
+#include <barretenberg/stdlib/recursion/verifier/verifier.hpp>
 
 using namespace proof_system::plonk;
```

Comment:
Aaaaah! Burn it! :)

Reply:
It'd be great if we could do a search for `using namespace` in this repo and obliterate all instances.

Reply:
Does it not get angry about `using namespace`?

Reply:
It does! But it doesn't have an auto-fix for them.

As we fix certain tidy-codes repo-wide we can start treating those as Errors.

Reply:
Commented on [this issue](https://github.com/AztecProtocol/aztec3-circuits/issues/190#issuecomment-1528816996) which should address this and then promote the corresponding code (`google-build-using-namespace`) to an error.

I changed our `.clang-tidy` a bunch after your comments here so that we flag pretty much every tidy-warning as an error except for the warning codes ones we explicitly omit from `WarningsAsErrors`. This way when we do a manual refactor that resolves such an issue project-wide, we can then easily promote it to an error.